source/slang/slang-check-expr.cpp

```diff
@@ -1286,39 +1286,39 @@ namespace Slang
                             // that takes a single `int` parmaeter and means to call that instead.
                             //
                             // For now that should be a non-issue, and in a pinch such a user
-                            // could use `T(0)` instead of `(T) 0` to get around this special
```

Comment:
The tabbing here seems to have changed - or been removed? We use 4 spaces for for each indent (we don't use tabs).